spec/system/placements/schools/placements/remove_a_placement_spec.rb

```diff
@@ -0,0 +1,101 @@
+require "rails_helper"
+
+RSpec.describe "Placements / Schools / Placements / Remove a placement",
+               type: :system, service: :placements do
+  let!(:placement_1) { create(:placement, school:, subjects: [subject_1]) }
+  let(:placement_2) { create(:placement, school:, subjects: [subject_2]) }
+  let(:school) { build(:placements_school, name: "School 1", phase: "Primary") }
+  let(:subject_1) { build(:subject, name: "Subject 1", subject_area: "primary") }
+  let(:subject_2) { build(:subject, name: "Subject 2", subject_area: "primary") }
+
+  before do
+    given_i_sign_in_as_anne
+    placement_2
+    when_i_visit_the_placement_show_page(placement_1)
+  end
+
+  scenario "User removes a placement from a school" do
+    and_i_click_on("Remove placement")
+    then_i_am_asked_to_confirm(school, placement_1, subject_1)
+    when_i_click_on("Cancel")
+    then_i_return_to_placement_page(school, placement_1)
+    when_i_click_on("Remove placement")
+    then_i_am_asked_to_confirm(school, placement_1, subject_1)
+    when_i_click_on("Remove placement")
+    then_the_placement_is_removed_from_the_school(school, placement_1, "Subject 1")
+    and_a_school_placement_remains_for_subject("Subject 2")
+  end
+
+  private
+
+  def and_there_is_an_existing_user_for(user_name)
+    user = create(:placements_user, user_name.downcase.to_sym)
+    user_exists_in_dfe_sign_in(user:)
+    create(:user_membership, user:, organisation: school)
+  end
+
+  def and_i_visit_the_sign_in_path
+    visit sign_in_path
+  end
+
+  def and_i_click_sign_in
+    click_on "Sign in using DfE Sign In"
+  end
+
+  def when_i_visit_the_placement_show_page(placement)
+    visit placements_school_placement_path(school, placement)
+  end
+
+  def given_i_sign_in_as_anne
+    and_there_is_an_existing_user_for("Anne")
+    and_i_visit_the_sign_in_path
+    and_i_click_sign_in
+  end
+
+  def given_a_placement_has_one_subject(placement, placements_subject)
+    PlacementSubjectJoin.create!(placement:, subject: placements_subject)
+  end
```

Comment:
This can be removed now.

Reply:
resolved in [646bdb8](https://github.com/DFE-Digital/itt-mentor-services/pull/334/commits/646bdb8b359916bdad8be58abb16d9054b7c50ac)